client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriverAuth.java

```diff
@@ -256,6 +256,36 @@ public void testSuccessFullSslVerification()
         }
     }
 
+    @Test
+    public void testSuccessFullSslVerificationAlternateHostname()
+            throws Exception
+    {
+        String accessToken = newJwtBuilder()
+                .setSubject("test")
+                .setHeaderParam(KEY_ID, "33")
+                .signWith(privateKey33)
+                .compact();
+
+        String url = format("jdbc:trino://127.0.0.1:%s", server.getHttpsAddress().getPort());
+        Properties properties = new Properties();
+        properties.setProperty("SSL", "true");
+        properties.setProperty("SSLVerification", "FULL");
```

Comment:
Can you please tests with for `hostnameInCertificate` with `SSLVerification=CA` and `SSLVerification=NONE`?

Reply:
yes! Added more logic as we only want this to be enabled for FULL ssl verification.